en/user_guide/keyword_management.rst

```diff
@@ -0,0 +1,13 @@
+Keyword management
+==================
+
+Open Orchestra allows to attach keywords to content or media. They describe the content and allow to manage it.
+In the keyword panel, you can create or destroy a keyword. If you destroy it, the keyword will stay in content type that already using it.
+
+Usage
+-----
+
+When you fill the meta data for a content or media, already created keyword can be chosen from a list.
+Alternatively, you can create new ones using the ``keyword`` field.
```

Comment:
can you also provide a screenshot of the difference between a created an existing keyword